assets/projects-single.css

```diff
@@ -0,0 +1,19 @@
+#readingArea h2 {
+    margin-top: 2rem;
+    margin-bottom: 0.6rem;
+}
+
+#readingArea figure img {
+    margin-top: 1em;
+    border-radius: 5px;
+}
+
+/* Small devices (landscape phones, 576px and up) */
+@media (min-width: 576px) {
+    .project-metadata {
+        display: flex;
+        gap: 1em;
+        flex-wrap: wrap;
+    }
+}
+
```

Comment:
as far as i can tell, all of the above can be written using bs. May u justify these pls?

Reply:
@AARandom Ok understood. But u can do it this way:

![image](https://github.com/user-attachments/assets/41cd399b-207b-4d02-9b51-246ab66c4869)